frontend/hub/namespaces/HubNamespacePage/HubNamespaceCLI.tsx

```diff
@@ -0,0 +1,54 @@
+import { useParams } from 'react-router-dom';
+import React from 'react';
+import { LoadingPage } from '../../../../framework';
+import ExternalLinkAltIcon from '@patternfly/react-icons/dist/esm/icons/external-link-alt-icon';
+import { Trans } from 'react-i18next';
+import { ClipboardCopy, PageSection } from '@patternfly/react-core';
+import { useGet } from '../../../common/crud/useGet';
+import { HubItemsResponse } from '../../useHubView';
+import { HubNamespace } from '../HubNamespace';
+import { hubAPI } from '../../api/formatPath';
+import { HubError } from '../../common/HubError';
+import { getRepoURL } from '../../api/utils';
+import styled from 'styled-components';
+
+export function HubNamespaceCLI() {
+  const repositoryUrl = getRepoURL('published');
+  const params = useParams<{ id: string }>();
+  const {
+    data: response,
+    error,
+    refresh,
+  } = useGet<HubItemsResponse<HubNamespace>>(
+    hubAPI`/_ui/v1/namespaces/?limit=1&name=${params.id ?? ''}`
+  );
```

Comment:
Is this needed? `data` is never used to show anything to the user. If it's not needed there's no need for `const params = useParams<{ id: string }>();` either.